### ISSUE TITLE
Stuck on step 14 (four blinks) of nx-on-wake installation (endless turn on turn off loop)

### ISSUE THREAD

**Issue by kasod** [ottokiksmaler/nx500_nx1_modding#122]
I've installed nx-on-wake on my NX1 but I think I sort of bricked my camera in that I'm perpetually stuck on the LED blinking 4 times and then shutting off and turning on again. I've tried deleting the file on the SD card but it still loops, just that the LED blinks only once instead of 4.

My NX1 did have an issue in that it was very sluggish in every mode except in special mode (it started behaving that way 6 months ago).  For instance, it usually took 10 seconds or more for it auto focus and take pictures or access the gallery and menu, but everything worked perfectly normal in special mode. Very weird but it is a 9-year-old camera. It's the only reason why I decided to mod it, as reinstalling firmware versions didn't help. The "good" thing is the mod actually seemed to fix it; the problem being that I can only experience the speed boost for a second before the camera turns off again. I'm guessing the sluggish manual mode of my NX1 messed up the installation somehow. 

Is there any recourse?

**Comment by vasile-gh** [comment]
The good news is that the camera seems to react to the content of the sd card.

Please try this:
- create a folder called nx-on-wake on the sd card.
- create a file in that folder called on-wake
- its content should be one line:


rm /opt/usr/nx-on-wake/on-wake

- put the sd card in your camera

Power up

Let me know what happened, if anything.

NB. You might need to use  /bin/rm instead of rm, I don't remember where rm is stored... 

Good luck...

**Comment by kasod** [comment]
Wow thank you for the reply. I didn't think anyone would actually see this! 

Yes it does seem to react to the sd card but sometimes the camera shut down faster than the LED can complete the four blinks, so sometimes you only see 1 one long blink and 2 fast ones. I think the forced shutdown is baked into the hardware now.

Path (sd card):
L:\nx-on-wake\on-wake
Content of the on-wake file:
`rm /opt/usr/nx-on-wake/on-wake`
or 
`/bin/rm /opt/usr/nx-on-wake/on-wake`

Unfortunately it didn't do anything. But hey vasile-gh, do you want the camera? It's actually in excellent condition externally (I tend to baby all my equipment). I have no use for it now. You've put in so much work into this project, it's kind of awe inspiring really. It's yours if you can pay for the shipping.

**Comment by vasile-gh** [comment]
Have you tried to ask Samsung to fix it? Maybe they still have some service capabilities for it...

Where are you based?

regards

**Comment by kasod** [comment]
I'm in Washington, US. I just figured bricked = unrepairable so asking Samsung didn't even cross my mind lol. I couldn't find any Samsung approved camera repair shops but I've just emailed a [local camera repair shop](https://www.glazerscamera.com/pages/repairs), so fingers crossed!

**Comment by kasod** [comment]
I just came to the realization that my firmware version might've been 1.40 thus the installation going haywire. Ugh. I think I rolled back just to keep the dev mode on my NX1. So yeah not the camera's fault.

**Comment by gtstud** [comment]
The repo is not outdated - it contains the most recent available version of the hack.

**Comment by vasile-gh** [comment]
The repo - AFAIK - contains the most recent versions of the various hacks.

**Comment by vasile-gh** [comment]
Yeah I think it was related to language settings (if memory serves me right). It was also for an earlier version of the mod.

Please leave it on M - I did not get any reports about the script not working IF you follow the steps to the letter. Furthermore, I did my mod following the script and it worked :-)...
